### PR TITLE
Onboarding redirect

### PR DESCRIPTION
**Pull request by ChrisHG-Ticketer**
Added functionality to retrieve URL for Hosted Onboarding Pages

### CODE REVIEW

**Comment by AlexandrosMor** [comment]
Hello @ChrisHG-Ticketer, 

Thank you for the pull request. We are going to validate your code and let you know as soon as possible 

kind regards,
Alexandros

File: Adyen/Model/Hop/GetOnboardingUrlRequest.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//  *                               ############
+//  *
+//  * Adyen Dotnet API Library
+//  *
+//  * Copyright (c) 2020 Adyen B.V.
+//  * This file is open source and available under the MIT license.
+//  * See the LICENSE file for more info.
+//  */
+#endregion
+
```
Comment (AlexandrosMor) [inline Adyen/Model/Hop/GetOnboardingUrlRequest.cs]:
```suggestion
using System;
using System.Text;
using Newtonsoft.Json;
```

**Comment by rikterbeek** [comment]
HI @ChrisHG-Ticketer,

Again thanks for the Pull Request are you able to change the code to the suggestions of @AlexandrosMor then we can merge the code.

Regards,
Rik
Adyen

File: Adyen/Model/Hop/GetOnboardingUrlResponse.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+//  *
+//  * Copyright (c) 2020 Adyen B.V.
+//  * This file is open source and available under the MIT license.
+//  * See the LICENSE file for more info.
+//  */
+#endregion
+
+using System.Runtime.Serialization;
+using Adyen.Model.MarketPay;
+namespace Adyen.Model.Hop
```
Comment (AlexandrosMor) [inline Adyen/Model/Hop/GetOnboardingUrlResponse.cs:26]:
```suggestion
using System.Collections.Generic;
using System.Text;
using Newtonsoft.Json;
namespace Adyen.Model.Hop
```

**Pull request merged**